ops/docker-compose.yml

```diff
@@ -47,7 +47,7 @@ services:
     # override with the dtl script and the env vars required for it
     entrypoint: ./dtl.sh
     env_file:
-        ./envs/dtl.env
+      -  ./envs/dtl.env
```

Comment:
I think the indent is off here

Reply:
Its technically valid syntax with or without the ident for lists, i was following the way that it is done in this file already to maintain the same syntax, the `services.xyz.ports` is indented